### PR TITLE
Update README.md

### LINKED ISSUE #19: [DOCS] Include technologies to the ReadMe
### Description

There needs information about next.js in the ReadMe.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by divyanshraj0408**
## Fixes Issue

closes #19 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by CBID2** [comment]
@divyanshraj0408, don’t forget to link the issue that your PR addresses. It’ll add more context about your PR :)

**Review by CBID2 — APPROVED** [review]
Looks good in my opinion.

**Comment by divyanshraj0408** [comment]
> @divyanshraj0408, don’t forget to link the issue that your PR addresses in the Fixes issue section. It’ll add more context about your PR :)
Thanks for letting @CBID2  me know, I totally missed it :+1:

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <img width="458" alt="Screenshot 2023-12-13 at 09 27 37" src="https://github.com/EddieHubCommunity/RepoRater/assets/624760/ccc20975-4788-4232-b9e8-c3356b387b32">
 
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
+### This project uses Next.js.
```
Comment (eddiejaoude) [inline README.md:6]: Thank you, this does not need to be a heading though

```suggestion

## Technologies used

- NextJS
- Appwrite
- DaisyUI

```

**Comment by eddiejaoude** [comment]
Thank you 👍

**Pull request merged**